general/echo/kmdf/driver/DriverSync/Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "echo-2"
 version = "0.1.0"
 license = "MIT OR Apache-2.0"
+rust-version = "1.72.1"
```

Comment:
This may be overly restrictive (since its latest stable), but I think its fine for the the samples repo. We can determine a lower-bound for the `windows-drivers-rs` crates later using [`cargo-msrv`](https://foresterre.github.io/cargo-msrv/index.html)

Reply:
I was on 1.70.0 and had some bindgen issues, I updated which got 1.72.1 so added that in to help make sure others can build the samples as easily as possible.